### ISSUE TITLE
How to pick `tok2vec` model for further use?

### ISSUE THREAD

**Issue by chopeen** [explosion/spaCy#6027]
Running `spacy pretrain` produces a number of models (1000, by default) and each of them is saved. The documentation does not explain how to choose the one for model training later:

- the last one?
- the one with lowest `Loss`?
- the one with lowest `Total Loss`?

Please document what one should do with all the models once `spacy pretrain` completes.

## Which page or section is this issue related to?

CLI / Pretrain: https://spacy.io/api/cli#pretrain

Sample output:
```
ℹ Not using GPU
✔ Created output directory: models/pretrain
✔ Saved settings to config.json
✔ Loaded input texts
✔ Loaded model 'en_core_web_lg'

============== Pre-training tok2vec layer - starting at epoch 0 ==============
  #      # Words   Total Loss     Loss    w/s
  0       208089   208617.438   208617   8090
  0       416523   411813.266   203195   7997
  0       630873   613573.500   201760   9112
  0       842845   804681.359   191107   9279
  0      1043900   975679.109   170997   9120
  0      1252988   1141108.80   165429   9178
  0      1467371   1300102.66   158993   9200
  0      1673788   1446423.95   146321   9137
  0      1873395   1582061.00   135637   9234
  0      2082116   1719134.95   137073   9112
  (...)
  1     14340391   7825318.44    87987   8316
  1     14557197   7917284.34    91965   8433
  1     14764661   8004568.09    87283   8384
  1     14970056   8091230.09    86662   8260
  1     15187211   8182793.30    91563   8390
  1     15385872   8266644.41    83851   8314
  1     15603092   8357522.61    90878   8341
  1     15817757   8446930.76    89408   8356
  1     16028249   8534980.90    88050   8468
  1     16217906   8613499.31    78518   8315
✔ Successfully finished pretrain
```

**Comment by svlandeg** [comment]
IIRC, the "total loss" is the loss accumulated across all epochs, so if I'm not mistaken that would be continuously increasing? In which case it's not a great predictor.

I think your best bet is to take the model with the lowest "loss", which represents the loss for that specific epoch (the output terminology could be improved here...). In general, you would expect the first few epochs to have relatively high losses, then it would start decreasing. It may keep decreasing, in which case your "best" model is the last. But if there is a "turning point" at a given moment, you should probably choose the model that was saved before the loss started increasing again.

**Comment by chopeen** [comment]
`Total Loss` does increase continuously. I have no idea how I could've missed it!

Should I submit a PR for the documentation page summing up what you wrote?

**Comment by svlandeg** [comment]
Yep, sure, let's try to summarize it in 1-2 sentences if it's not already clear from the documentation :-)
